nbs/common.base_windows.ipynb

```diff
@@ -177,10 +177,19 @@
     "            windows = windows.reshape(-1, window_size, len(temporal_cols))\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #186.            # print('temporal_data', temporal_data.max())
</pre><p>Are this prints necessary ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/301/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
thanks for catching that